pkg/resources/syslogng/statefulset.go

```diff
@@ -272,6 +272,16 @@ func (r *Reconciler) bufferMetricsSidecarContainer() *corev1.Container {
 					MountPath: BufferPath,
 				},
 			},
+			Resources: corev1.ResourceRequirements{
+				Limits: corev1.ResourceList{
+					corev1.ResourceMemory: resource.MustParse("10M"),
+					corev1.ResourceCPU:    resource.MustParse("1m"),
+				},
+				Requests: corev1.ResourceList{
+					corev1.ResourceMemory: resource.MustParse("5M"),
+					corev1.ResourceCPU:    resource.MustParse("1m"),
+				},
```

Comment:
I would lean towards not setting a cpu limit in general, but we do that everywhere, so at least set it to something that allows the container to peak a little bit if needed to avoid slow scrape times.

For memory, let's keep request/limit the same, to avoid surprises in general, [see](https://twitter.com/muhammeddanis7/status/1587190572879446016)
```suggestion
				Limits: corev1.ResourceList{
					corev1.ResourceMemory: resource.MustParse("10M"),
					corev1.ResourceCPU:    resource.MustParse("50m"),
				},
				Requests: corev1.ResourceList{
					corev1.ResourceMemory: resource.MustParse("10M"),
					corev1.ResourceCPU:    resource.MustParse("1m"),
				},
```

Reply:
sounds good to me, done in b68e60adea95bb053946c011f411205cd935b10f